ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Client.scala

```diff
@@ -320,7 +320,7 @@ private[ember] class H2Client[F[_]](
 
 private[ember] object H2Client {
   private type TinyClient[F[_]] = Request[F] => Resource[F, Response[F]]
-  def impl[F[_]: Async](
+  def impl[F[_]: Async: Network](
```

Comment:
Yet several minor releases of fs2, one will be able to implement the NIO framework merely via capabilities! 👍🏻